### ISSUE TITLE
Why do people still avoid writing tests?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/practical-full-stack-testing#37]
I ask myself this question all the time!

**Comment by nelsonic** [comment]
In my *experience* its:
+ Misconception of what TDD _**is**_ - many people _still_ think testing is for "_testers_"

**Comment by iteles** [comment]
Also, people think it will take them ages. They prefer the [single marshmallow now](https://en.wikipedia.org/wiki/Stanford_marshmallow_experiment) because they don't really understand they'll get two later :wink:

**Comment by nelsonic** [comment]
Agreed. But how do we phrase that...?

**Comment by freak4pc** [comment]
For me it just felt like a time-consuming tasks. Startups don't always tend to be able to create tests for their servers/products.

**Comment by nelsonic** [comment]
@freak4pc agree. it _does_ feel time-consuming at the _start_ of a project to have to setup a Testing Framework and Continuous Integration/Delivery System, but the good news is that you can get started in less than one hour with _existing **free**_ tools that make the process ~~painless~~ _pleasurable_!

**Comment by freak4pc** [comment]
Do you already have a document describing this? 
I would love to write some tests for my node.js server and my iOS Apps.

**Comment by nelsonic** [comment]
@freak4pc what are the elements of your Node.js stack? (is it "_MEAN_" or are you using something else?)

**Comment by freak4pc** [comment]
not using Mongo or Angular so it's not Mean.

RethinkDB & Redis over Express

**Comment by nelsonic** [comment]
@freak4pc  _**Love it**_ !! :heart:
We're using a _similar_ stack for our project(s) and in this Book, _except_ we've swapped Express for _**Hapi.js**_ ... 
Right now our tutorials are separate:
+ **Hapi**: https://github.com/nelsonic/learn-hapi
+ **Redis**: https://github.com/docdis/learn-redis
+ **ElasticSearch**: https://github.com/docdis/learn-elasticsearch
+ **Intro to Test Driven Development** (TDD): https://github.com/nelsonic/learn-tdd
+ **Travis**: https://github.com/docdis/learn-travis
+ **Istanbul**: https://github.com/docdis/learn-istanbul
+ **SauceLabs**: https://github.com/docdis/learn-saucelabs
+ **Riot** (_work-in-progress_): https://github.com/nelsonic/learn-riot
+ **Rethink** (_work-in-progress_) https://github.com/nelsonic/learn-rethinkdb
etc...
But we will be turning these into a cohesive book with a set of mini-projects which people will learn how to build.

**Comment by freak4pc** [comment]
I'm still not sure where to start @nelsonic . Where could I learn how to write some basic tests with Mocca/Chai for Node / Express for example ? How do I start?
